compiler/rustc_interface/src/callbacks.rs

```diff
@@ -35,7 +35,6 @@ fn track_diagnostic(diagnostic: &mut Diagnostic, f: &mut dyn FnMut(&mut Diagnost
             if let Some(diagnostics) = icx.diagnostics {
                 let mut diagnostics = diagnostics.lock();
                 diagnostics.extend(Some(diagnostic.clone()));
```

Comment:
this should be doable in a single `diagnostics.lock().extend(...)` expression, right?

Reply:
👍